go.mod

```diff
@@ -8,7 +8,7 @@ replace k8s.io/klog => github.com/istio/klog v0.0.0-20190424230111-fb7481ea8bcf
 
 replace github.com/spf13/viper => github.com/istio/viper v1.3.3-0.20190515210538-2789fed3109c
 
-replace github.com/gogo/protobuf => github.com/istio/gogo-protobuf v1.2.2-0.20190726125433-4c9abdb3090c
+replace github.com/gogo/protobuf => github.com/istio/gogo-protobuf v1.2.2
```

Comment:
Does this not force the usage of v1.2.2 of our tag?

Reply:
I think yes.